pkg/apiserver/webhooks/datavolume-mutate.go

```diff
@@ -97,8 +98,24 @@ func (wh *dataVolumeMutatingWebhook) Admit(ar admissionv1.AdmissionReview) *admi
 		targetName = ar.Request.Name
 	}
 
+	modifiedDataVolume := dataVolume.DeepCopy()
+	modified := false
+
+	config, _ := wh.cdiClient.CdiV1beta1().CDIConfigs().Get(context.TODO(), common.ConfigName, metav1.GetOptions{})
```

Comment:
Maybe return an error here like in DataSource's GET above?
https://github.com/kubevirt/containerized-data-importer/blob/9a4061c997220d1ae707006d76a079a7450ff330/pkg/apiserver/webhooks/datavolume-mutate.go#L85-L88

Reply:
Sure will